br/pkg/task/common.go

```diff
@@ -609,9 +606,6 @@ func (cfg *Config) ParseFromFlags(flags *pflag.FlagSet) error {
 		return errors.Trace(err)
 	}
 
-	if cfg.Checksum, err = flags.GetBool(FlagChecksum); err != nil {
```

Comment:
if we decide to have two separate configs to control checksum, i.e. `backupChecksum`, `restoreChecksum`, I guess we need to parse them both in their respective cfg so code can actually use. 

the flow is like
1. parse flags if it's from cmd.
2. build cfg either from cmd flags or directly in brie
3. pass cfg to backup/restore